### ISSUE TITLE
Stuck in Google Sign in loop

### ISSUE THREAD

**Issue by manuelarte** [Iakobs/godot-google-play-game-services-android-plugin#14]
Hi,

I have the following code in my `_ready` function in my main scene:

```gdscript
func _ready() -> void:
	### Authenticate with Google Play Services ###
	if OS.has_feature("android"):
		SignInClient.is_authenticated()
		SignInClient.user_authenticated.connect(func(is_authenticated: bool): # (1)
			if not is_authenticated:
				SignInClient.sign_in()
		)
	###
	...
```

But when I try to run my game in my android device I am stuck in a infinite loop of asking to authenticate, am I doing something wrong?

**Comment by Iakobs** [comment]
Hi @manuelarte I was taking a break for holidays, sorry for being so late! I see you closed the issue the same day, I hope that's because you solved it. Please let me know if you have any more questions :)

**Comment by manuelarte** [comment]
Hi @lakobs, I closed it because this is for Godot 3.5, and I'm having the issue with Godot 4.
